include/circt/Dialect/Datapath/Datapath.td

```diff
@@ -0,0 +1,21 @@
+//===- Datapath.td - Datapath dialect definition -----------*- tablegen -*-===//
```

Comment:
This emacs tag is not necessary any more: https://github.com/llvm/llvm-project/commit/e2c3d16282aca0f7e13f58170140647632d627c9